apps/src/templates/projects/FeaturedProjectsTable.jsx

```diff
@@ -82,7 +84,10 @@ export const styles = {
 // Cell formatters.
 const thumbnailFormatter = function (thumbnailUrl, {rowData}) {
   const projectUrl = `/projects/${rowData.type}/${rowData.channel}/`;
-  thumbnailUrl = thumbnailUrl || PROJECT_DEFAULT_IMAGE;
+  thumbnailUrl =
+    thumbnailUrl ||
+    PROJECT_DEFAULT_THUMBNAIL_IMAGE_OVERRIDE[rowData.type] ||
+    PROJECT_DEFAULT_IMAGE;
```

Comment:
I see that this code is basically re-used down in `ProjectCard.jsx`. Could it be broken out into a utility function?

maybe as

```
const thumbnailUrl = getThumbnailUrl(path, type)

const getThumbnailUrl = (path, type) =>
  path
  || PROJECT_DEFAULT_THUMBNAIL_IMAGE_OVERRIDE[rowData.type] 
  || PROJECT_DEFAULT_IMAGE
```

where `getThumbnailUrl` is defined in some external file with access to those `PROJECT_DEFAULT_*` constants.

Then this function becomes `thumbnailUrl = getThumbnailUrl(thumbnailUrl, rowData.type)` (could also re-name the parameter so it's not re-assigned...`const thumbnailUrl = getThumbnailUrl(givenThumbnailUrl, rowData.type)`, maybe?)

And then `ProjectCard` lines 100-104 becomes `getThumbnailUrl(projectData.thumbnailUrl, type)`

There may be a better name for it.

I'm just worried about needing to keep these two bits of code in sync and forgetting to update one or the other down th eline.

Reply:
Sounds good. Added function to a new `projectUtils.ts` file. Thanks!